monai/data/utils.py

```diff
@@ -15,6 +15,7 @@
 from itertools import starmap, product
 from torch.utils.data._utils.collate import default_collate
 import numpy as np
+from scipy.ndimage.filters import gaussian_filter
```

Comment:
MONAI already has PyTorch based [gaussian_filter](https://github.com/Project-MONAI/MONAI/blob/master/monai/networks/layers/simplelayers.py#L38).
And we are trying to remove the dependency on scipy, so please use our own gaussian filter and generate data on expected device directly.
Thanks.

Reply:
Took a look at it. It can't be used in current state. There is no support for a sequence of sigmas which is essential here.

Reply:
Hi @wyli ,

As you are working on the ticket to remove the `scipy.ndimage` dependency, do you think it's possible to enhance our gaussian filter for this usage case?
If using our own gaussian filter, `importance_map` can be CUDA Tensor directly, which is more efficient.
Thanks.